.gitignore

```diff
@@ -24,3 +24,4 @@ node_modules/
 
 #flattened POMs
 .flattened-pom.xml
+package-lock.json
```

Comment:
the `package-lock.json` needs to be checked in and must not be ignored.

Reply:
Ah, I had assumed package-lock.json was something to protect concurrent execution (like a pid file). Just learned what it *really* is.